ValveResourceFormat/Resource/ResourceTypes/Morph.cs

```diff
@@ -0,0 +1,215 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using ValveResourceFormat.IO;
+using ValveResourceFormat.Serialization;
+using ValveResourceFormat.Serialization.KeyValues;
+using ValveResourceFormat.Serialization.NTRO;
+
+namespace ValveResourceFormat.ResourceTypes
+{
+    public class Morph
```

Comment:
It would be better to make extend ` : KeyValuesOrNTRO` so that it's automatically constructed for MRPH / morf files in Resource.cs. (You probably looked at Mesh, which also doesnt it currently, but I want to fix that).

It already has ` nameof(BlockType.MRPH) => new KeyValuesOrNTRO(BlockType.MRPH, "MorphSetData_t"),`, so that can be upgraded to use this class.

If you need help or have questions, feel free to hop on discord.